Hazel/src/Hazel/Debug/Instrumentor.h

```diff
@@ -183,7 +213,7 @@ namespace Hazel {
 	#define HZ_PROFILE_BEGIN_SESSION(name, filepath) ::Hazel::Instrumentor::Get().BeginSession(name, filepath)
 	#define HZ_PROFILE_END_SESSION() ::Hazel::Instrumentor::Get().EndSession()
 	#define HZ_PROFILE_SCOPE(name) ::Hazel::InstrumentationTimer timer##__LINE__(name);
-	#define HZ_PROFILE_FUNCTION() HZ_PROFILE_SCOPE(HZ_FUNC_SIG)
+	#define HZ_PROFILE_FUNCTION() HZ_PROFILE_SCOPE(::Hazel::Clean_Expression(HZ_FUNC_SIG).data)
```

Comment:
I am still debating with myself, should the cleanup be the part outside the macro, or should the cleanup be part of the `HZ_FUNC_SIG` itself?
```cpp
#define HZ_FUNC_SIG __FUNCSIG__
#define HZ_PROFILE_FUNCTION() HZ_PROFILE_SCOPE(::Hazel::Clean_Expression(HZ_FUNC_SIG).data)
```
versus
```cpp
#define HZ_FUNC_SIG ::Hazel::Clean_Expression(__FUNCSIG__).data
#define HZ_PROFILE_FUNCTION() HZ_PROFILE_SCOPE(HZ_FUNC_SIG)
```

Reply:
I think the place where it is right now is the perfect spot

Reply:
HZ_FUNC_SIG changes depending on the used compiler - so we should wrap the CleanExpression function around that.

Reply:
Done.